### ISSUE TITLE
Inconsistency in graph controls

### ISSUE THREAD

**Issue by blacklightpy** [obsidian-html/obsidian-html#608]
In the local graph view, right click selects the node, and left click loads the page. In global graph view, right click loads the page whereas left click switches the node.

This is applicable for both 2d and 3d graphs.

**Comment by dwrolvink** [comment]
Yes this was done on purpose because I'd figure that you'd want to explore the structure in the full page mode and follow links in the small view under a note.

I'm guessing you'd want to ve able to configure them to be the same?

**Comment by blacklightpy** [comment]
Yup. Even though I can understand this for myself, I feel like this will be difficult to explain to visitors. Also, I too often prefer to explore in the small view.

**Comment by dwrolvink** [comment]
![image](https://github.com/obsidian-html/obsidian-html/assets/30291690/bbd4bc79-ce16-4fd8-8f57-3de922e807be)

Added text to instruct users to the capabilities and now the full page and in-page graphs work the same.
